### PR TITLE
Also map name to last_name

### PR DESCRIPTION
**Pull request by Liberatys**
(phylor) You correctly pointed out in #6  that last_name is not a valid attribute ^^ I'd like to keep last_name as an accessible attribute on the entry model as I find it easier to read/include than just 'name'. 

(I also added the required KEY for Development to README)

### CODE REVIEW

File: spec/system/simple_query_spec.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
     it "returns a response with the informaton for the address" do
       entries = subject.entries
       expect(entries[0].name).to eq("Farnell AG")
+      expect(entries[0].last_name).to eq("Farnell AG")
```
Comment (phylor) [inline spec/system/simple_query_spec.rb:15]: @Liberatys Do we want `last_name` only to be set for `type == Person`? I'd personally slightly prefer that, but I'm totally fine with setting it to the company name, if that is your intention!

Comment (Liberatys) [reply spec/system/simple_query_spec.rb:15]: sounds like a good idea 👍🏻 will limit it to type == Person

Comment (Liberatys) [reply spec/system/simple_query_spec.rb]: I figured if we set last_name for a person we can also set company for an organisation :D

**Review by phylor — DISMISSED** [review]
Great! :+1: :medal_sports:

**Pull request merged**